Docs: Add cURL examples to folder_dashboard_search.md (#26461)
grafana/grafana: docs/sources/http_api/folder_dashboard_search.md

  }
]
```
## cURL examples

> The most basic example for a dashboard for which there is no authentication required is:

Reviewer: This is not a general case: thee reason that there is no authentication required in the above example is because in the play site is enabled [the anonymous access](https://grafana.com/docs/grafana/latest/auth/#anonymous-authentication).

Also I'm not sure it's generally good to include an example of a request which does not require authentication towards our play site.

Author: Should I just remove the whole example? Can you suggest a better one?

Reviewer: I would say yes. The easier way to perform API requests is using the credentials with basic authentication which is already included.
The alternative would be to suggest to enable anonymous access and try:
`curl http://localhost:3000/api/search` but it's no more a basic example.

Whatever you think is better.

Author: Updated. Please make sure what I wrote is correct.

> ```curl

Reviewer 2: I don't think it's correct to use curl as language specifier for code blocks, since it's not a language. The language here would be a form of shell script, f.ex. bash. See [GitHub reference](https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks#syntax-highlighting).
```suggestion
```
```

> curl 'https://play.grafana.org/api/search'

Reviewer: ```suggestion
curl https://play.grafana.org/api/search
```

```

> Here’s a cURL command that works for getting the home dashboard when you are running Grafana locally with default admin credentials:
> 

Reviewer: ```suggestion
Here’s a cURL command that works for getting the home dashboard when you are running Grafana locally when [basic authentication](https://grafana.com/docs/grafana/latest/http_api/auth/#basic-auth) is enabled using the default admin credentials:

```

> ```curl

Reviewer 2: ```suggestion
```
```

curl http://admin:admin@localhost:3000/api/dashboards/home
```

> 

Reviewer: ```suggestionA

For all the available ways to make authenticated requests to the API please check the [Authentication API]({{<relref "../auth"}}) section.
```
